app/views/scenarios/_form.html.erb

```diff
@@ -13,12 +13,12 @@
 
     <div class="field">
       <%= f.label :game_system %><br>
-      <%= f.select :game_system, %w(PFS SFS Other) %>
+      <%= f.select :game_system, %w(PFS SFS Playtest PFS2 Other) %>
```

Comment:
As long as there are separate game systems, would it make more sense to classify cards as a separate one? Don't need to do it here, especially if it complicates stuff.